feconf.py

```diff
@@ -132,6 +132,9 @@
 ENTITY_TYPE_QUESTION = 'question'
 ENTITY_TYPE_VOICEOVER_APPLICATION = 'voiceover_application'
 
+VOICEOVER_APPLICATION_REVIEW = 'review'
```

Comment:
These constant names don't really make sense. Normally constants are named as {{CONSTANT_TYPE}}_{{SPECIFIC_LABEL}}.

Looking at where these are used -- it seems odd to condition things on a particular "purpose". Can you create separate handlers instead for the two use cases?

Reply:
I've addressed these comments in the handler PR at #7936.